Sources/PackageGraph/ModulesGraph+Loading.swift

```diff
@@ -35,7 +35,9 @@ extension ModulesGraph {
         customXCTestMinimumDeploymentTargets: [PackageModel.Platform: PlatformVersion]? = .none,
         testEntryPointPath: AbsolutePath? = nil,
         fileSystem: FileSystem,
-        observabilityScope: ObservabilityScope
+        observabilityScope: ObservabilityScope,
+        productsFilter: ((Product) -> Bool)?,
+        modulesFilter: ((Module) -> Bool)?
```

Comment:
Should we default in the public interface?

Reply:
good point, updated now